### PR TITLE
Check if mac when displaying editor mode toggle shortcut

### PR DESCRIPTION
**Pull request by vladanost**
## Description

Related to https://github.com/WordPress/gutenberg/pull/3755
Detecting Mac when displaying editor mode toggle shortcut.

## How Has This Been Tested?

Tested on Mac and Linux.

## Types of changes

New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: editor/edit-post/keyboard-shortcuts.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
+// eslint-disable-next-line
+const isMac = navigator.platform.toUpperCase().indexOf( 'MAC' ) >= 0;
```
Comment (youknowriad) [inline editor/edit-post/keyboard-shortcuts.js]: I guess you could do `window.navigator` to avoid the eslint issue?

**Review by youknowriad — APPROVED** [review]
Thanks 👍

**Comment by aduth** [comment]
If this file is consumed by the `KeyboardShortcuts` (or `Mousetrap` generally) we can use the `mod` keyword to automatically toggle Command / Ctrl.

https://craig.is/killing/mice

**Comment by youknowriad** [comment]
@aduth This is for "display" purpose. To show the actual shortcut in the ellipsis menu.

**Pull request merged**